sdk/monitor/Azure.Monitor.Query/api/Azure.Monitor.Query.netstandard2.0.cs

```diff
@@ -58,6 +78,25 @@ public enum ServiceVersion
             V2023_05_01_PREVIEW = 1,
         }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct MetricsQueryAudience : System.IEquatable<Azure.Monitor.Query.MetricsQueryAudience>
```

Comment:
Is it expected that we would define a new audience struct per library? Seems like there should be a common one defined in Azure.Core.
/cc @tg-msft @m-nash

Reply:
This is the path we have taken across our SDKs for now. I believe at some point we had conversation on unifying things in azure-core but I don't think we got to a consensus.

Reply:
Should we at least consolidate to a single struct per library?

Reply:
Good question! I thought the audience was per client because audience value and support can differ per client however I do not know what precedence we have used here in the past or if this happens to be the first one.

Reply:
I think it's hard to use a single struct for all of monitor because the endpoints can differ - and we have a mix of data-plane and management endpoints.

Reply:
What about [ArmEnvironment](https://learn.microsoft.com/en-us/dotnet/api/azure.resourcemanager.armenvironment?view=azure-dotnet)?